metrics/examples/prometheus/kubernetes.yaml

```diff
@@ -53,6 +64,16 @@ data:
           - source_labels: [__meta_kubernetes_pod_name]
             action: replace
             target_label: kubernetes_pod_name
+      - job_name: 'kafka_service_job'
```

Comment:
So this is basically an example file, right? What is the point of using both pods and services in the same config file and scraping from both? I though it should be either one or another.

Reply:
The jobs have different names, so you can draw multiple graphs and compare them. If you do think this is useless, I can remove redundant jobs.

Reply:
Well ... I'm not expert ... but it seems strange to collect the metrics twice. I think that if we want to show both examples, it should be in two separate config files as in _one or the other_. Users do not really want to compare both sources, right? They probably just want the one source they choose to work :-)